src/threshold-sigs/base-ot.ts

```diff
@@ -20,10 +20,10 @@ export class Responses extends BytearrayWrapper {}
 
 export class HashedKeys extends BytearrayWrapper {}
 
-export class Participant {
-  // Id of this participant
+export class BaseOTParticipant {
```

Comment:
This is probably my lack of understanding here, but I don't see this class `BaseOTParticipant` used anywhere in the tests or in other places in the code.

Reply:
In threshold sig [test](https://github.com/docknetwork/crypto-wasm-ts/pull/38/files#diff-a41053a6293b3a8d52f9171ad6373ebc831c7ca3bec9e48dd21d7dd3a66e0d6c).

Reply:
Now, I see. thanks! 👍